docker/package/baking_wizard.py

```diff
@@ -0,0 +1,303 @@
+            "sudo systemctl start tezos-node-" + self.config["network"] + ".service"
+        )
+
+        return_code = None
+        while return_code != 0:
+            os.system("sleep 5")
+            return_code = subprocess.call(
+                shlex.split("sudo -u tezos tezos-client bootstrapped")
+            )
+
+    # Importing the baker key
+    def import_baker_key(self):
+        if self.config["key_import_mode"] == "secret-key":
+            proc_call(
+                "sudo -u tezos tezos-client import secret key baker"
+                + self.config["secret_key"]
+            )
+        elif self.config["key_import_mode"] == "json":
+            proc_call(
+                "sudo -u tezos tezos-client activate account baker with"
+                + self.config["json_filepath"]
+            )
+
+        else:
+            print()
+            input("Please open the Tezos Wallet app on your ledger, then hit Enter.")
+            proc_call(
+                "sudo -u tezos tezos-client import secret key baker"
+                + self.config["ledger_url"]
+            )
+
+            print()
+            input("Please open the Tezos Baking app on your ledger, then hit Enter.")
+            proc_call("sudo -u tezos tezos-client setup ledger to bake for baker")
+
+    def register_baker(self):
+        proc_call("sudo -u tezos tezos-client register key baker as delegate")
+        print(
+            "You can check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/) "
+            "to see the baker status and baking rights of your account."
+        )
+
+    def start_baking(cfg):
+        proc_call("sudo systemctl start tezos-baking-" + cfg["network"] + ".service")
+
+
+def main():
+    cfg = run_wizard(["mainnet", "florencenet"])
+
+    if cfg is not None:
+        readline.set_completer(None)
+        setup = Setup(cfg)
+
+        print("Trying to bootstrap tezos-node")
+        setup.bootstrap_node()
+
+        if cfg["mode"] == "node":
+            return 0
+
+        print("Importing the baker key")
+        setup.import_baker_key()
+
+        print("Registering the baker")
+        setup.register_baker()
+
+        print("Start baking instance")
+        setup.start_baking()
+
+        print("Checking the services status")
+        setup.check_systemd_status()
+
+        print(
+            "Congratulations! All required Tezos infrastructure services should now be started."
+            " You can show logs for all the services using the 'tezos' user by running: "
+            "journalctl -f _UID=$(id tezos -u)"
+        )
+
+        if cfg["mode"] == "baker":
+            print(
+                "To stop the baking instance, run: sudo systemctl stop tezos-baking"
+                + cfg["network"]
+                + ".service"
+            )
+
+
+if __name__ == "__main__":
+    readline.parse_and_bind("tab: complete")
+
+    try:
+        main()
+    except:
+        print("Exiting Tezos Baking Wizard")
```

Comment:
Let's stop all started services in case the wizard exited with an error. My setup failed with baker import and I was confused by the fact that node was running after the failure